scopes/lanes/ui/overview/menu/lanes-overview-menu.tsx

```diff
@@ -83,7 +91,7 @@ function TopBarNav(props: NavLinkProps) {
     <NavLink
       {...props}
       className={classnames(props.className, styles.topBarLink)}
-      activeClassName={classnames(props.className, styles.active)}
+      activeClassName={classnames(props.className, target === pathname && styles.active)}
```

Comment:
seems you're passing the same className from props both to the base className and to activeClassName. this is a bad idea. because anything you pass to the non active will collide with the active styles

Reply:
I would remove the `props.className` from the activeClass and replace with `props.activeClassName`
```
activeClassName={classnames(props.activeClassName, target === pathname && styles.active)}
```